app/views/spree/admin/orders/new.html.haml

```diff
@@ -1,14 +1,14 @@
 - content_for :page_title do
-  = t(:new)
+  = t(:new_order)
+  \#
+  = @order.number
 
 - content_for :page_actions do
   %li= button_link_to t(:back_to_orders_list), spree.admin_orders_path, :icon => 'icon-arrow-left'
 
 = admin_inject_shops(module: 'admin.orders')
 = admin_inject_order_cycles
 
-= render 'spree/admin/shared/order_tabs', :current => 'Order Details'
```

Comment:
Are we purposefully removing the order tabs?

Reply:
no, needs to be re-added.

Reply:
done